Assets/Scripts/SkinCardDisplayer.cs

```diff
@@ -0,0 +1,87 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Security.Cryptography;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class SkinCardDisplayer : MonoBehaviour
+{
+    [SerializeField] private Skins skins;
+    [SerializeField] private int skinCountPerRow;
+    [SerializeField] private GameObject cardsPrefab;
+    [SerializeField] private HorizontalLayoutGroup cardsGroup;
+    [Space] 
+    [SerializeField] private ChangeFrame increaseFrame;
+    [SerializeField] private ChangeFrame decreaseFrame;
+    
+    public int frame;
+
+    public int SkinCountPerRow => skinCountPerRow;
+
+    private void Start()
+    {
+        frame = 0;
+        SpawnCards();
+        Debug.Log(PlayerPrefs.GetString("skin"));
+    }
+
+    private void OnEnable()
+    {
+        increaseFrame.onIncreaseFrame += IncreaseFrame;
+        decreaseFrame.onDecreaseFrame += DecreaseFrame;
+    }
+    
+    private void OnDisable()
+    {
+        increaseFrame.onIncreaseFrame -= IncreaseFrame;
+        decreaseFrame.onDecreaseFrame -= DecreaseFrame;
+    }
+
+    private void SpawnCards()
+    {
+        RemoveCards();
+        for (var i = frame; i < frame + skinCountPerRow; i++)
+        {
+            if(i<skins.SkinCards.Count)
+                CreateCard(skins.SkinCards[i]);
+        }
+    }
+
+    private void CreateCard(SkinCard skinCard)
+    {
+        var card = Instantiate(cardsPrefab, Vector2.zero, Quaternion.identity);
+        card.transform.SetParent(cardsGroup.transform);
+        
+        card.GetComponent<SkinCardController>().SetCard(skinCard.skinSprite,skinCard.name);
+        
+        if(skinCard.skinSprite.name == PlayerPrefs.GetString("skin"))
```

Comment:
make this boolean a function, what does this do?

Reply:
Nothing That much important, I wanted to display selected card with red color, but I have to find better solution

Reply:
ok find a better solution